### PR TITLE
 control plan machines can not be deleted

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to 4.6+ versions
- [Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=1940773)
- [Preview](http://file.del.redhat.com/asakthiv/BZ1940773/machine_management/deleting-machine.html)
-@zhsun ptal

### CODE REVIEW

**Review by darshan-nagaraj — COMMENTED** [review]
LGTM. 
I have suggested a minor fix. I would request peer-review-squad to review my comments.

File: modules/machine-delete.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@
 [id="machine-delete_{context}"]
 = Deleting a specific machine
 
-You can delete a specific machine.
+You can delete a specific machine but it is important to note that you can not delete a control plane machine.
```
[the remark this answers is not in the document]
Comment (Amrita42) [reply modules/machine-delete.adoc:10]: I am not sure if this is an assumption that we should make, lets wait for a second opinion.

[the remark this answers is not in the document]
Comment (Amrita42) [reply modules/machine-delete.adoc:10]: Agree with **cannot**

**Comment by jboxman-rh** [comment]
@Amrita42 @darshan-nagaraj 

So I might also add to an _Additional resources_ section of the assembly file a link to the document on how to delete a control plane node as well, which is linked to here: https://bugzilla.redhat.com/show_bug.cgi?id=1940773#c5

In case someone does want to delete a control plane node specifically.

Thanks!

File: modules/machine-delete.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (Amrita42) [reply modules/machine-delete.adoc:10]: @jboxman whats your perspective on whether a note should be added or include as part of a sentence.

[the remark this answers is not in the document]
Comment (Amrita42) [reply modules/machine-delete.adoc:10]: Its a suggestion but I am not sure that we can include it as a note or part of the sentence.

File: machine_management/deleting-machine.adoc [hunk 2]
```diff
@@ -9,3 +9,10 @@ toc::[]
 You can delete a specific machine.
 
 include::modules/machine-delete.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information, see   xref:../backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.adoc[Replacing an unhealthy etcd member].
```
Comment (kalexand-rh) [inline machine_management/deleting-machine.adoc:16]:
```suggestion
* xref:../backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.adoc[Replacing an unhealthy etcd member].
```

> Each item in the unordered list must contain a minimum of text besides the link or [xref](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc).

File: machine_management/deleting-machine.adoc [hunk 3]
```diff
@@ -9,3 +9,8 @@ toc::[]
 You can delete a specific machine.
 
 include::modules/machine-delete.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
```
Comment (jboxman-rh) [inline machine_management/deleting-machine.adoc:14]: @Amrita42, sorry to hold up this short PR, but I just realized that we're using `=` for _Additional resources_ in assemblies, per the contributing guidelines:

> Additionally, in an assembly, use == formatting for the section heading (== Additional resources). Use of this heading syntax at the assembly level indicates that the sections relate to the whole assembly. For example:

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections

Looks like an easy change, and then we can get this merged, thanks!

Comment (Amrita42) [reply machine_management/deleting-machine.adoc]: Thanks @jboxman-rh , Done.

Comment (jboxman-rh) [reply machine_management/deleting-machine.adoc]: @Amrita42, also requires an `id` such as:

```
[id="additional-resources_configuring-alert-notifications"]
```

per the guidelines. Thanks!

Comment (Amrita42) [reply machine_management/deleting-machine.adoc]: @jboxman-rh done..

**Comment by jboxman-rh** [comment]
@Amrita42 thanks!

**Pull request merged**